Source/ACE.Server/Network/GameAction/Actions/GameActionAcceptTrade.cs

```diff
@@ -6,6 +6,13 @@ public static class GameActionAcceptTrade
         [GameAction(GameActionType.AcceptTrade)]
         public static void Handle(ClientMessage message, Session session)
         {
+            uint partnerGuid = message.Payload.ReadUInt32();
+            ulong tradeStamp = message.Payload.ReadUInt64();
+            uint tradeStatus = message.Payload.ReadUInt32();
+            uint initiatorGuid = message.Payload.ReadUInt32();
+            bool initatorAccepts = message.Payload.ReadBoolean();
```

Comment:
is the structure of this packet confirmed?

this would be something unseen before in ac protocol: whenever a boolean is sent, it's usually sent as an int

tradeStamp - have never seen a field sent as 64bit before

the total packet size here would also be 162 bits, which would need an .Align() to get it back in sync with the 4-byte dword boundary